src/pages/settings/settings.ts

```diff
@@ -123,6 +200,21 @@ export class SettingsPage {
     });
   }
 
+  ionViewDidLeave() {
+    if (this.bitpayIdRef) {
+      this.bitpayIdRefSubscription.unsubscribe();
```

Comment:
What happen if I close the app in this view? The subscription will remain active. Adds "Platform.pause" to catch that event when device enter on background.